Makefile

```diff
@@ -9,6 +9,7 @@ OCID_INSTANCE := ocid_dev
 SYSTEM_GOPATH := ${GOPATH}
 PREFIX ?= ${DESTDIR}/usr
 INSTALLDIR=${PREFIX}/bin
+GO_MD2MAN ?= /usr/bin/go-md2man
```

Comment:
worth adding `go-md2man` to `make install.tools` - feel free to skip this comment, I'll make a follow up maybe